pytest_mozwebqa/pytest_mozwebqa.py

```diff
@@ -199,13 +192,6 @@ def pytest_addoption(parser):
     config = ConfigParser.ConfigParser(defaults={'baseurl': ''})
     config.read('mozwebqa.cfg')
```

Comment:
Is it intentional to have both a `mozwebqa.cfg` and a `setup.cfg`? Is that because the former will be committed with the repo and the latter is meant to be local only? Might it be simpler for users if there was only one `.cfg` file with each of the different sections?